services/server/server_linux.go

```diff
@@ -67,3 +69,8 @@ func apply(ctx context.Context, config *srvconfig.Config) error {
 func newTTRPCServer() (*ttrpc.Server, error) {
 	return ttrpc.NewServer(ttrpc.WithServerHandshaker(ttrpc.UnixSocketRequireSameUser()))
 }
+
+func applyBoltOpenOptions(options *bolt.Options) {
+	options.MmapFlags = syscall.MAP_POPULATE
```

Comment:
Is it possible to make it configurable? `MAP_POPULATE` will touch all the pages, which might take long time to be ready. I don't have data about normal size of metadata db. So I would like to make it configurable.

Reply:
Will put up a separate PR for this. Thanks!